### DISCUSSION TITLE
Duplicate mails in sent folder

### DISCUSSION THREAD

**Discussion by tim-web** [the-djmaze/snappymail#1743] [Q&A] [answered]
Hi Folks, 
On a brand new install (first time using) of snappymail:latest from docker, I am seeing duplicate emails in my sent items folder.  This only happens when the draft is saved/autosaved.  The mail provider is zoho and this seems to be a common issue, it's resolved on other mail clients by disabling saving the message to sent items.

When sending via SMTP, zoho will save a copy in the senders SMTP folder.  If the mail client also saves a copy directly in the IMAP folder, then we end up with a duplicate.

I'm not sure if this is a bug or a feature I can't find...
Feature ? =>   Is there a way in snappymail to disable saving messages to sent items?
Bug ? =>  The behaviour is inconsistent when a message is saved.  No save it works as expected; saved I end up with duplicates.

Thanks

**Comment by the-djmaze** [comment]
Draft should not be (auto)saved in Sent folder.
You should use something like:
![afbeelding](https://github.com/user-attachments/assets/fb6306a3-8ed6-4dac-9545-4577b189ef3d)

**Reply by tim-web** [reply, marked as answer]
The folders are configured as such.   I don't have any theories of why doubles only show up in the sent folder when a draft has been saved.

On the double copies of the sent email, the Zoho saved copy has a SMTP received-by header, whereas the copy saved by snappymail in the sent folder has no additional headers.

As a workaround, I have set the sent system folder mapping to blank/none.   This prevents snappymail from saving a copy of the email in the folder, so I only end up with the copy saved by the mail provider zoho.

**Reply by the-djmaze** [reply]
When it works , it works 👍
Good workaround for Zoho, you found.
